client/packages/system/src/Name/apiModern/useContactsGraphQL.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  useAuthContext,
+  useGql,
+  useQueryClient,
+} from '@openmsupply-client/common';
+import { getSdk } from './operations.generated';
+
+export const useContactGraphQL = () => {
+  const { client } = useGql(); // generates client connection
+  const queryClient = useQueryClient(); //
+  const { storeId } = useAuthContext(); // manages auth, grabs store id
+  const contactApi = getSdk(client); // auto generates the code to call our graphql layer in the server (what happens when you run yarn generate)
```

Comment:
can drop the comments. Just because the other files like so doesn't have comments. Keep consistency